### PR TITLE
CI-1456 --no-timestamp flag for adcm_sdk_path

### PR DESCRIPTION
**Pull request by mooncube**

### CODE REVIEW

File: src/bin/adcm_sdk_pack [hunk 1]
```diff
@@ -31,14 +31,24 @@ if __name__ == "__main__":
                         help='set splitter release flag to true', default=None)
     parser.add_argument('-e', '--edition', default=None, required=False,
                         help='edition to build, usefull for spec with multiple editions')
+    parser.add_argument('-s', '--no-timestamp', action="store_true",
```
Comment (dgusakov) [inline src/bin/adcm_sdk_pack:34]: Provide a default value

Comment (dgusakov) [reply src/bin/adcm_sdk_pack:34]: And add required=False

Comment (mooncube) [reply src/bin/adcm_sdk_pack:34]: Added for clarity, but really 'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.

**Review by dgusakov — DISMISSED** [review]

File: src/adcm_client/packer/data/config_data.py [hunk 2]
```diff
@@ -61,13 +62,21 @@ def _from_file(self, key, **kwargs):
...
-            value = [entry.get(key) for entry in self.data
-                     if (entry.get('type') == 'cluster'
-                         or entry.get('type') == 'provider'  # noqa: W503
-                         or entry.get('type') == 'host')]  # noqa: W503
+            value = [
+                entry.get(key)
+                for entry in self.data
+                if (
+                    entry.get('type') == 'cluster'
+                    or entry.get('type') == 'provider'  # noqa: W503
```
Comment (Maunty) [inline src/adcm_client/packer/data/config_data.py:70]:
```suggestion
                    or entry.get('type') == 'provider'
```

we disabled flake here and moved to black, so there is no need to inline flake commands.

File: src/adcm_client/packer/data/config_data.py [hunk 3]
```diff
@@ -61,13 +62,21 @@ def _from_file(self, key, **kwargs):
...
+                    or entry.get('type') == 'host'
+                )
+            ]  # noqa: W503
```
Comment (Maunty) [inline src/adcm_client/packer/data/config_data.py:73]:
```suggestion
            ]
```
same here

**Review by Maunty — CHANGES_REQUESTED** [review]

File: src/adcm_client/packer/data/config_data.py (hunk 3 above)
Comment (mooncube) [reply src/adcm_client/packer/data/config_data.py]: Fixed.

**Review by dgusakov — APPROVED** [review]

**Pull request merged**